package.json

```diff
@@ -16,76 +7,57 @@
     "type": "git",
     "url": "https://github.com/skatejs/skatejs"
   },
-  "main": "dist/index.js",
-  "jsnext:main": "src/index.js",
+  "main": "lib/index.js",
```

Comment:
This is the default configuration. I'd rather just keep it as is because it's less opinion for us to have to worry about. If it changes, we update the `package.json`.